b2b-sample/guardio-insurance-sample-application/util/util/apiUtil/decodeUser.js

```diff
@@ -25,7 +25,8 @@ export default function decodeUser(user) {
     return {
         "id": user.id,
         "username": user.userName,
-        "name": user.name != undefined ? user.name.givenName : "Not Defined",
-        "email": user.emails != undefined ? user.emails[0] : "Not Defined"
+        "firstName": user.name != undefined ? user.name.givenName : "-",
+        "familyName": user.name != undefined ? user.name.familyName : "-",
+        "email": user.emails != undefined ? user.emails[0] : "-"
     };
-}
+}
```

Comment:
Let's add a new line at the end of the file.

Reply:
Fixed with https://github.com/Achintha444/samples-is/commit/4120b50f44ba393f00f7c7cf57aa04d21f6bc8d6#diff-63d09dc68fe77aadebf84c960e9e5bd8b971f4e692ac00404169d3487ae387a5